### PR TITLE
2.9.0/sdk improvements part 1

### PR DESCRIPTION
**Pull request by MusiikoAS**

### CODE REVIEW

File: android/src/main/java/com/inbrain/InBrainSurveysModule.java [hunk 1]
```diff
@@ -45,15 +45,10 @@
 public class InBrainSurveysModule extends ReactContextBaseJavaModule implements InBrainCallback {
 
     private final ReactApplicationContext reactContext;
-    private HashMap sessionData;
```
Comment (susslik) [inline android/src/main/java/com/inbrain/InBrainSurveysModule.java:48]: Looks like you forgot about `public void setSessionID(final String sessionId)` and `public void setDataOptions(final ReadableMap data)` methods - they still uses those variables. Please, update the methods.
And do not forget to run the app at both platforms (iOS and Android) to ensure all is fine...

**Review by susslik — CHANGES_REQUESTED** [review]
Hey @MusiikoAS - I've added a few comments, could you check them pls when you're free?

File: src/InBrain.tsx [hunk 2]
```diff
@@ -167,21 +185,26 @@ const setOnCloseListener = (callback: () => void) => {
  */
 const setOnCloseListenerFromPage = (callback: () => void) => {
     onCloseFromPage = callback;
-}
+};
 
 /**
- * Validation for options.
+ * Validation for apiClientId and apiSecret.
  * TODO: find any validation library
```
[the remark this answers is not in the document]
Comment (MusiikoAS) [reply src/InBrain.tsx:192]: it's not my commend :)

File: src/InBrain.tsx [hunk 3]
```diff
@@ -167,21 +185,26 @@ const setOnCloseListener = (callback: () => void) => {
...
-    assertNotNullNorEmpty("apiClientId", apiClientId)
-    assertNotNullNorEmpty("apiSecret", apiSecret)
+const validateCliendData = (apiClientId: string, apiSecret: string) => {
+    assertNotNullNorEmpty("apiClientId", apiClientId);
+    assertNotNullNorEmpty("apiSecret", apiSecret);
+};
 
+/**
+ * Validation for options.
+ * TODO: find any validation library
```
[the remark this answers is not in the document]
Comment (MusiikoAS) [reply src/InBrain.tsx:201]: it's not my commend :)

File: src/InBrain.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (susslik) [reply src/InBrain.tsx:192]: I know, looks like this comment was added a looong time ago, however it's useless now

File: src/InBrain.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (susslik) [reply src/InBrain.tsx:201]: I know, looks like this comment was added a looong time ago, however it's useless now

**Pull request merged**